app/controllers/sessions_controller.rb

```diff
@@ -13,7 +14,15 @@ def create
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     redirect_to root_path
   end
+
+  def remember_check checked, user
+    if checked == Settings.CHECKED
+      remember user
+    else
+      forget user
+    end
```

Comment:
dùng toán tử ngôi số 3 có bị quá 80 digit ko chú
`(checked == Settings.CHECKED) ? remember(user) : forget(user)`